### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Casprovy**

### CODE REVIEW

File: src/components/client/Basket.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ showBasket = () => {
...
 }
 
+checkoutZero = () => {
+  let check = this.totalPrice();
+  if (check===0) {
+    return 'disabled' }
+      else {
+     return '';
+      }
+    }
```
Comment (oliversisson) [inline src/components/client/Basket.js]: why not
```
return (this.totalPrice() === 0) ? 'disabled' : '';
```

File: src/components/client/cards/checkout-form.js [hunk 2]
```diff
@@ -13,36 +13,40 @@ class CheckoutForm extends Component {
...
   }
 
-  render() {
-    if (this.state.complete) return <p className="black-text">Purchase Complete</p>;
+  message = () => {
+  if (this.state.complete) { return <div><p className="black-text">Purchase Complete</p></div>;
+  } else {
+    return <div><p className="black-text">Would you like to complete the purchase?</p></div>;
+  }
+  }
```
Comment (oliversisson) [inline src/components/client/cards/checkout-form.js]: nit: indent

Comment (oliversisson) [reply src/components/client/cards/checkout-form.js]: also, consider ternary

**Comment by Casprovy** [comment]
Thanks!

**Pull request closed without merging**